apps/src/music/context.ts

```diff
@@ -9,13 +9,18 @@ type PlayerUtils = Pick<
   'getPlaybackEvents' | 'getTracksMetadata' | 'getLengthForId' | 'getTypeForId' | 'getLastMeasure'
 >;
 
+interface Other {
+  getIsExecutingPlay: () => boolean
+};
```

Comment:
Looking at this in context, it may just be easier to stick this in its own context (IsPlayingContext?) so we don't have to overload PlayerUtils. MusicView can just pass the value directly. I'm thinking that this might be useful in general to expose the is playing state to all children in a readonly manner.

Reply:
I'll add a new `ExecutionContext` though will probably need some improvements.